src/test/java/htsjdk/variant/variantcontext/writer/VCFWriterUnitTest.java

```diff
@@ -26,10 +26,13 @@
 package htsjdk.variant.variantcontext.writer;
```

Comment:
Should also test interaction between `setVcfHeader()` and `writeHeader()` (ie., test calling one and then the other for the various implementation).